### PR TITLE
Split wait and run into separate steps

### PR DESCRIPTION
**Pull request by lanwen**
This change brings some additional params, like `args` `wait` `logfile` as well as improves cosmetics and adds tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,10 @@ jobs:
       - name: Build with Maven
         run: mvn -V -B verify
 ```
+
+## Configurable parameters
+
+- `token` (__required__):  `secret-string` - service account token for Testcontainers Cloud
```
Comment (eddumelendez) [inline README.md:39]:
```suggestion
- `token` (__required__):  `secret-string` - service account token for Testcontainers 
Cloud
- `version` (__optional__):  `latest` (default) - Testcontainers Cloud binary version
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+name: TCC GitHub Action
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+jobs:
+  defaults:
+    runs-on: ubuntu-latest
```
Comment (eddumelendez) [inline .github/workflows/test.yml:9]: I think we can add the macos and windows and make sure it works on all of them

Comment (lanwen) [reply .github/workflows/test.yml:9]: done

**Review by eddumelendez — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**